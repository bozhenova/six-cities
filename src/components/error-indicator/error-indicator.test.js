import React from 'react';
import ErrorIndicator from './error-indicator';
import renderer from 'react-test-renderer';

describe('Error Indicator component renders correctly', () => {
  it('should render an error message', () => {
    const tree = renderer.create(<ErrorIndicator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
