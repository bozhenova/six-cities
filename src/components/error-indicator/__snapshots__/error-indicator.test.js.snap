// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Indicator component renders correctly should render an error message 1`] = `
<div
  className="center"
>
  Something's gone wrong
</div>
`;
